Packages/doc/databrowser_paplot.rst

```diff
@@ -0,0 +1,223 @@
+.. _db_paplot:
+
+Pulse Average (PA) plot
+=======================
+
+When the stimulus set contains multiple pulses the pulse average plot allows to
+extract these single pulses to overlay, and average and visually inspect these.
+
+Although the stimulus set will contain :ref:`wavebuilder_pulsetrain` epochs in
+most cases, the pulse extraction algorithm works with all epoch types and
+special measurement modes like :ref:`daephys_dDAQ` or :ref:`daephys_oodDAQ`.
+
+The dDAQ slider from the :ref:`db_settings` tab is respected as is the :ref:`db_channelselection`.
+
+For instructional purposes assume you have acquired a pulse train with oodDAQ on 4 different headstages.
+This means you have 4 active headstages with each a DA and AD channel. We
+only visualize the AD data in the PA plot.
+
+The PA plot will then have 16 = 4 x 4 PA sets (either in one big graph or in multiple graphs).
+
+Each set will have one source of pulse starting times (called "region")
+and one source of the visualized data. The pulse starting times are always
+extracted from the DA channel of the region.
+
+That means the top left set is from the first region and the first
+active headstage, the one to the right is from the second region and the
+first active headstage. Or to put it differently, regions are the columns.
+
+The following two figures, :ref:`Figure Pulse Average Trace plot` and
+:ref:`Figure Pulse Average Image plot`, were created with multiple sweeps
+overlayed, see :ref:`db_overlaysweeps`, and time alignment and zeroing turned on.
+
+.. _Figure Pulse Average Trace plot:
+
+Trace plot
+""""""""""
+
+.. figure:: PulseAverage_traces.png
+   :align: center
+   :width: 75%
+
+   Trace plot
+
+The trace plot is a display of all single pulses from a PA set as traces. The
+scale bars, shown in black, give an impression of the data range. The trace
+colors coincide with the trace colors used in the databrowser and denote the
+headstage of the data, see also :ref:`Figure Relevant Colors`.
```

Comment:
The trace plot displays data in each PA set as voltage or current time-series. Scale bars, shown in black, are provided for each y axes and for each column's shared x (time) axes. PA trace colors match databrowser trace colors and encode the headstage, see also :ref:`Figure Relevant Colors`.